tasks/kernel_matrix_testing/ci.py

```diff
@@ -146,7 +146,7 @@ def get_vm(self, distro: str, vmset: str) -> Optional[Tuple[str, str]]:
                     return microvm['id'], microvm['ip']
         return None
 
-    def get_vm_boot_log(self, distro: str, vmset: str) -> Optional[str]:
+    def get_vm_boot_log(self, distro: str, vmset: str) -> str | None:
```

Comment:
This syntax is not valid in Python 3.9 which is the minimum supported version, should we updated the python version ? (cf README.md)

Reply:
This is valid as long as you have this import `from __future__ import annotations`. The linter is running on Python 3.8 in the circleci image

Reply:
ftr, this works for python 3.7+, so no need to modify the python version in the readme